package.json

```diff
@@ -60,12 +62,14 @@
     "babel-jest": "^24.7.1",
     "babel-loader": "^8.0.5",
     "babel-preset-env": "^1.7.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-plugin-import": "^2.17.1",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.12.4",
+    "commander": "^3.0.1",
+    "eslint": "^6.7.1",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.16.0",
     "jest": "^24.7.1",
+    "json-schema-faker": "github:json-schema-faker/json-schema-faker#master",
```

Comment:
Can we use pinned version instead of master? I have got problem `npm i`: npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/qingyashu/Repos/guppy-server/node_modules/json-schema-faker/bin/gen.js'

Reply:
could you pin this to `"json-schema-faker": "0.5.0-rc23"`? thanks